test/javascripts/turbolinks_test.coffee

```diff
@@ -163,6 +174,12 @@ describe 'Turbolinks', ->
           assert(Turbolinks.fullPageNavigate.called, 'Should perform a full page refresh.')
           done()
 
+      it 'changed assets do not update the browser history stack', (done) ->
```

Comment:
when it fails in the console you see

`Turbolinks head asset tracking with turbolinks-track="true" changed assets do not update the browser history stack`

which I think is pretty understandable